lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/AddressSessionService.java

```diff
@@ -60,6 +79,13 @@ public AddressSessionService(
         this.clock = clock;
     }
 
+    public ValidationResult<ErrorObject> validateTokenRequest(TokenRequest tokenRequest) {
+        if (!tokenRequest.getAuthorizationGrant().getType().equals(GrantType.AUTHORIZATION_CODE)) {
```

Comment:
the api gateway is already doing this validation - personally I think we should have integration tests that check this rather than a unit test